tests/test_acasclient.py

```diff
@@ -1797,4 +1595,299 @@ def test_044_author_and_role_apis(self):
         with self.assertRaises(requests.HTTPError) as context:
             acas_user_author_role['userName'] = test_username
             user_client.update_author_roles(author_roles_to_delete=[acas_user_author_role])
-        self.assertIn('500 Server Error', str(context.exception))
\ No newline at end of file
+        self.assertIn('500 Server Error', str(context.exception))
+
+
+
+def experiment_load_test(data_file_to_upload, dry_run_mode, self):
```

Comment:
This function uses `self` strangely. Can we move this function into TestExperimentLoader or BaseAcasClientTest and move self to the beginning so it is called like `self.experiment_load_test(data_file_to_upload, True)`? That would be more pythonic

Reply:
Refactored, thanks.